pennylane/compiler/qjit_api.py

```diff
@@ -111,3 +112,170 @@ def circuit(theta):
     compilers = AvailableCompilers.names_entrypoints
     qjit_loader = compilers[compiler]["qjit"].load()
     return qjit_loader(fn=fn, *args, **kwargs)
+
+
+def while_loop(*args, **kwargs):
+    """A :func:`~.qjit` compatible while-loop for PennyLane programs.
+
+    .. note::
+
+        When used with :func:`~.qjit`, this function only supports
+        the Catalyst compiler. Please see
+        :func:`catalyst.while_loop` for more details.
```

Comment:
```suggestion
        the Catalyst compiler. See
        :func:`catalyst.while_loop` for more details.
```

Because I repeated 'please see' twice 😆